src/browser/modules/Stream/CypherFrame/RelatableView/relatable-view.tsx

```diff
@@ -80,6 +80,7 @@ export function RelatableViewComponent({
 
   return (
     <RelatableStyleWrapper>
+      {/* @ts-ignore */}
```

Comment:
I played around a little bit with this because it was odd to get a new error 🤔 

It seemed like the error was from a mismatch in types -> you got the exact version of react-table from relate-by-ui, but the @types/react-table are version `7.0.16`, when I changed it to be the same as the react-table `7.0.4` the problem went away!

Reply:
I get some other ts errors when updating to that version.

Reply:
Looks like I was a bit quick to say it worked.. Let's keep it as is then